modules/compute/gke-node-pool/outputs.tf

```diff
@@ -16,23 +16,24 @@
 
 output "node_pool_name" {
   description = "Name of the node pool."
-  value       = google_container_node_pool.node_pool.name
+  value       = google_container_node_pool.node_pool[*].name
```

Comment:
is this always going to be an array? 
One possible direction here is to make sure it is, so that you don't have special cases. I would alsy rename the variable to node_pool_names and always return an array. Then you can change the other places that consume node_pool_name to consume arrays instead. WDYT? 
This would require you share this with the rest of your team and make sure they are ok, as it will change the "interface" of this and the other gke modules.

Reply:
Yes, it is an array / list of strings.

I will check on how to make sure the other places consume it as array. Started a thread with the team on this.